app/controllers/home_controller.rb

```diff
@@ -7,15 +7,17 @@ def show
     partial_dir = Rails.root.join("app", "views", "home")
     if user_signed_in?
       redirect_to stream_path
-    elsif is_mobile_device?
+    elsif is_mobile_device? || request.format == :mobile
```

Comment:
I don't know the body of the is_mobile_device method but shouldn't this test be included in it?